MsixCore/msixmgr/CommandLineInterface.cpp

```diff
@@ -479,4 +667,30 @@ void CommandLineInterface::SetWVDFileType(std::wstring fileType)
     {
         this->m_fileType = WVDFileType::NotSpecified;
     }
-}
\ No newline at end of file
+}
+
+std::wstring CommandLineInterface::GetFileTypeAsString()
+{
+    switch (this->m_fileType)
+    {
+        case WVDFileType::CIM: return L"CIM";
+        case WVDFileType::VHD: return L"VHD";
+        case WVDFileType::VHDX: return L"VHDX";
+        case WVDFileType::NotSpecified: return L"NotSpecified";
+    }
+}
+
+std::wstring CommandLineInterface::GetOperationTypeAsString()
+{
+    switch (this->m_operationType)
+    {
+    case OperationType::Add: return L"Add";

```

Comment:
I think it should return "AddPackage" and not just "Add". We are entering this data in DB. When we will be building dashboard, we can directly use this to display. If we insert "Add", then somewhere we will have to write this mapping that "Add" corresponds to "AddPackage" while creating dashboard.

Reply:
Changed as suggested